packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -103,9 +118,23 @@ class SessionReplay implements SessionReplayEnrichmentPlugin {
         void this.storeEventsForSession(this.events, this.currentSequenceId);
       },
       packFn: pack,
+      maskAllInputs: true,
```

Comment:
what does maskAllInputs do?

Reply:
pretty much what it says, see the guide: https://github.com/rrweb-io/rrweb/blob/master/guide.md, we'll be adding configuration to this down the line to allow customers to unmask some inputs, but legal asked that this be in place before we start dogfooding